code/controller/controller.h

```diff
@@ -19,6 +19,7 @@ class Controller {
     enum Type {
         ACTUATOR,
         SIMULATOR,
+        SOLENOID,

```

Comment:
I think we can just ditch `ACTUATOR` then?

Reply:
Yeah I want to slowly remove all the actuator files for now. Trying to remove everything at once proved to open a nest of bugs 😓